src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/utils.ts

```diff
@@ -43,17 +43,25 @@ export const THORCHAIN_AFFILIATE_NAME = 'ss'
 // BPS are needed as part of the memo, but 0bps won't incur any fees, only used for tracking purposes for now
 const AFFILIATE_BPS = 0
 
+const usdcEthereumAssetId: AssetId = 'eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48'
+const usdcAvalancheAssetId: AssetId =
+  'eip155:43114/erc20:0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e'
+export const usdtEthereumAssetId: AssetId =
+  'eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7'
+
 // The minimum amount to be sent both for deposit and withdraws
 // else it will be considered a dust attack and gifted to the network
 export const THORCHAIN_SAVERS_DUST_THRESHOLDS = {
   [btcAssetId]: '30000',
   [bchAssetId]: '10000',
   [ltcAssetId]: '10000',
   [dogeAssetId]: '100000000',
-  [ethAssetId]: '0',
-  [avalancheAssetId]: '0',
+  [ethAssetId]: '10000000000',
+  [avalancheAssetId]: '10000000000',
```

Comment:
still seeing this as 0 from https://thornode.ninerealms.com/thorchain/inbound_addresses, was this intentional?

Reply:
Absolutely is, was losing a sanity trying to figure out why withdraws weren't picked by THOR, turns out there's a 10 Gwei minimum we have to honor here https://github.com/thorswap/SwapKit/blob/46f2f2557ead060adde1e4d4ca5cd5efb1ec7bd3/packages/swapkit/swapkit-entities/src/entities/assetAmount.ts#L115

Strictly speaking, the dust amount is still 0 for ETH and AVAX, but I didn't want to bring outbound / dust threshold complexity here and instead add it here. Given this is ~ 0.000016 USD at the time of writing, I believe this won't block anyone from depositing !